app/src/main/java/com/kickstarter/viewmodels/ProjectCardHolderViewModel.java

```diff
@@ -0,0 +1,279 @@
+package com.kickstarter.viewmodels;
+
+
+import android.support.annotation.NonNull;
+
+import com.kickstarter.libs.ActivityViewModel;
+import com.kickstarter.libs.Environment;
+import com.kickstarter.libs.rx.transformers.Transformers;
+import com.kickstarter.libs.utils.BooleanUtils;
+import com.kickstarter.libs.utils.NumberUtils;
+import com.kickstarter.libs.utils.ObjectUtils;
+import com.kickstarter.libs.utils.ProgressBarUtils;
+import com.kickstarter.libs.utils.ProjectUtils;
+import com.kickstarter.models.Category;
+import com.kickstarter.models.Photo;
+import com.kickstarter.models.Project;
+import com.kickstarter.models.User;
+import com.kickstarter.ui.viewholders.ProjectCardViewHolder;
+
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+import rx.Observable;
+import rx.subjects.PublishSubject;
+
+import static com.kickstarter.libs.rx.transformers.Transformers.coalesce;
+
+public interface ProjectCardHolderViewModel {
+
+  interface Inputs {
+    /** Call to configure the view model with a project. */
+    void configureWith(Project project);
+    void projectClicked();
+  }
+
+  interface Outputs {
+    /** Emits the string name of the projects category. */
+    Observable<String> backersCountText();
+    Observable<Boolean> backingViewGroupIsGone();
+    Observable<String> categoryNameText();
+    Observable<String> deadlineCountdownText();
+    Observable<Boolean> featuredViewGroupIsGone();
+    Observable<List<User>> friendsForNamepile();
+    Observable<String> friendAvatarUrl();
+    Observable<Boolean> imageIsInvisible();
+    Observable<Boolean> friendBackingViewIsHidden();
+    Observable<Boolean> fundingUnsuccessfulTextViewIsGone();
+    Observable<Boolean> metadataViewGroupIsGone();
+    Observable<Project> projectOutput();
```

Comment:
this output has me a bit 🤔. tracing down how's it used I see it's due to the fact that our deadline helpers require a full project instead of a date https://github.com/kickstarter/android-oss/blob/888a3746835835016fc1a1bb32d8e2a90b8bffce/app/src/main/java/com/kickstarter/libs/utils/ProjectUtils.java#L126-L129. that's a bit 🤔 🤔.

if yer up for it, I say we disentangle that so that those helpers just take a date, and then this output can be more succinctly described as `projectDeadline` or something.